molo/profiles/templates/profiles/viewprofile.html

```diff
@@ -8,6 +9,9 @@
   <h3>{% trans "Hello " %}{{ user.username }}</h3>
   <label class="label-wide">{% trans "Display Name: " %}{{ user.profile.alias | default:'Anonymous' }}</label>
   <label class="label-wide">{% trans "Date of Birth: " %}{{ user.profile.date_of_birth | date:"M d, Y" }}</label>
+  {% if settings.profiles.UserProfilesSettings.show_mobile_number_field %}
+    <label class="label-wide">{% trans "Mobile Number " %}{{ user.profile.mobile_number }}</label>
+  {% endif %}
```

Comment:
Noting from our discussion so I don't forget: We should not render the mobile number if it is `None`.